### PR TITLE
Redo workflow

### LINKED ISSUE #1: Missing 2021 Playoffs Data
1. **Have you installed the latest development version of the package(s) in question? **

Currently Using nflreadr version 1.2.0.16
2. **Describe the bug**

2021 Season Playoffs data seems to be missing when loading data 
3. **Reprex**

```r

espnqbrdata<-load_espn_qbr("nfl",2021) 

```

4. **Expected Behaviour**

Only regular season data available for 2021, for other years playoffs is included

5. **nflverse_sitrep()**

<details>
  <summary> sitrep </summary>
  
  ```r
  # ℹ You've asked for the packages nflseedR, nfl4th, nflplotR, and nflverse which are not installed. 
They are skipped.
── System Info ─────────────────────────────────────────────────────────────────────────────────────
• R version 4.2.0 (2022-04-22 ucrt)   • Running under: Windows 10 x64 (build 19043)
── nflverse Packages ───────────────────────────────────────────────────────────────────────────────
• nflreadr (1.2.0.16)
• nflfastR (4.3.0   )
── nflverse Options ────────────────────────────────────────────────────────────────────────────────
No options set for nflreadr and nflfastR
── nflverse Dependencies ───────────────────────────────────────────────────────────────────────────
• cachem      (1.0.6)   • glue       (1.6.2)    • qs            (0.25.3)   
• cli         (3.3.0)   • janitor    (2.1.0)    • R6            (2.5.1)    
• codetools   (0.2-18)  • jsonlite   (1.8.0)    • RApiSerialize (0.1.0)    
• cpp11       (0.4.2)   • lattice    (0.20-45)  • rappdirs      (0.3.3)    
• crayon      (1.5.1)   • lifecycle  (1.0.1)    • Rcpp          (1.0.8.3)  
• curl        (4.3.2)   • listenv    (0.8.0)    • RcppParallel  (5.1.5)    
• data.table  (1.14.2)  • lubridate  (1.8.0)    • rlang         (1.0.2)    
• digest      (0.6.29)  • magrittr   (2.0.3)    • snakecase     (0.11.0)   
• dplyr       (1.0.9)   • Matrix     (1.4-1)    • stringfish    (0.15.7)   
• ellipsis    (0.3.2)   • memoise    (2.0.1)    • stringi       (1.7.6)    
• fansi       (1.0.3)   • mgcv       (1.8-40)   • stringr       (1.4.0)    
• fastmap     (1.1.0)   • nlme       (3.1-158)  • tibble        (3.1.7)    
• fastrmodels (1.0.2)   • parallelly (1.32.0)   • tidyr         (1.2.0)    
• furrr       (0.3.0)   • pillar     (1.7.0)    • tidyselect    (1.1.2)    
• future      (1.26.1)  • pkgconfig  (2.0.3)    • utf8          (1.2.2)    
• generics    (0.1.2)   • progressr  (0.10.1)   • vctrs         (0.4.1)    
• globals     (0.15.0)  • purrr      (0.3.4)    • xgboost       (1.6.0.1)  

  ```
</details>

6. **Screenshots**

7. **Additional context**

### LINKED ISSUE #2: missing qb data and dup obs
espn_Qbr_rating_21 <- load_espn_qbr("nfl", 2021, "weekly")

# WAS qb data missing week 1, 2021
# david mills dups obs week 7, 2021, likely other dups

### LINKED ISSUE #3: Overhaul automation
- [ ] Seasons are hard coded
- [ ] data released in git history instead of releases
- [ ] scripts depend on weekday they are running on
- [ ] POs sometimes missing

### PR DESCRIPTION
**Pull request by mrcaseb**
This PR

- removes unused code and data related to pfr adv stats,
- adds a new update qtr script that works with releases,
- updates the gh workflow for qbr

### CODE REVIEW

File: scrape-adv-stats-boxscores.R
Comment (tanho63) [inline scrape-adv-stats-boxscores.R:1]: If removing these files you may also want to remove the action
https://github.com/nflverse/espnscrapeR-data/blob/master/.github/workflows/update-adv.yaml

Comment (tanho63) [reply scrape-adv-stats-boxscores.R:1]: Also noticed you didn't tackle the FPI stuff that is automated here https://github.com/nflverse/espnscrapeR-data/blob/master/.github/workflows/update-fpi-pbwr.yaml

Comment (mrcaseb) [reply scrape-adv-stats-boxscores.R:1]: removed the pfr workflow

Comment (mrcaseb) [reply scrape-adv-stats-boxscores.R:1]: i did actually think about fpi and win rates but it seems like we don't provide any loader in nflreadr. And win rates aren't available beyond 2022

**Review by tanho63 — APPROVED** [review]

**Pull request merged**